Marlin/src/feature/pause.cpp

```diff
@@ -558,9 +558,17 @@ void resume_print(const float &slow_load_length/*=0*/, const float &fast_load_le
     thermalManager.reset_hotend_idle_timer(e);
   }
 
+  if (targetTemp > thermalManager.degTargetHotend(active_extruder))
+    thermalManager.setTargetHotend(targetTemp, active_extruder);
```

Comment:
Simplified, since the temp is only set when above target.

Reply:
Good catch.